GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -948,5 +971,62 @@ public void SwitchFocus(bool alreadyContainedFocus)
                 DiffFiles.Focus();
             }
         }
+
+        /// <summary>
+        /// Hotkey handler
+        /// </summary>
+        /// <returns>true if executed</returns>
+        private bool StageSelectedFiles()
+        {
+            var selectedIds = DiffFiles.SelectedItems.SecondIds().ToList();
+            if (!DiffFiles.Focused || selectedIds.Count != 1 || selectedIds.FirstOrDefault() != ObjectId.WorkTreeId)
+            {
+                return false;
+            }
```

Comment:
The meaning of the return value is rather "hotkey handled" than "command executed".
IIUIC, the return value is used to find another control which handles the same hotkey.
I would not support the reuse of these hotkeys.
So this check should be omitted and the function always returns true. Then we do not need a dedicated function.

Reply:
The check still need to be there, but return true (since early return is preferred rather than check if allowed).

Reply:
Ah OK. Then I would return nothing, i.e. void, and return true directly in `ExecuteCommand`.

Why must `DiffFiles` be focused?
The hotkeys will be reused for operations on selected lines in the `DiffText`.
I think it suffices to check whether `DiffText` has the focus. Then "selected lines" else "selected files".

Reply:
The logic here (with #7825 included) is the same as in FormCommit, so I propose to keep it and check for FileStatusList focus. 
In #7825 the method return false if it could not be executed, not sure what should be done here...
But I will change these functions to void as suggested.

Reply:
Changed so return false if FileStatusList is not focused only and consider the key consumed otherwise
Other keyhandling may cause both hotkey and script execution, should probably be changed.